### DISCUSSION TITLE
Disabling Certificate Requests on 0.14?

### DISCUSSION THREAD

**Discussion by rtkay123** [hyperium/hyper#2637] [Q&A] [answered]
Hi there, I'm just looking around and I see #947 
That's pretty dated and those functions aren't available in the recent versions of hyper. Where would I go from here:

```rs
//using the open ssl crate
 
        let mut context_builder = SslContextBuilder::new(SslMethod::tls())?;
        context_builder.set_verify(SslVerifyMode::NONE);
        let ssl_context = context_builder.build();
```

How would I set that context to my `hyper::client::Client`? I don't see anything I can work with in the `Client` [docs](https://docs.rs/hyper/0.14.12/hyper/client/struct.Builder.html)

**Comment by seanmonstar** [comment, marked as answer]
hyper no longer includes TLS pieces, since people have different requirements for what TLS library they use. You could make use of `hyper-tls`, `hyper-rustls`, `hyper-openssl`, or others to get an `HttpsConnector` to use with `hyper::Client`.

If you want "easy mode", you may find using the `reqwest` crate handles a lot of these decisions for you, and just exposes client config options.

**Reply by rtkay123** [reply]
Thanks for the heads up. I know where to look now. I'll give hyper a shot.

**Reply by bonedaddy** [reply]
@kawaki-san did you ever figure out how to do this? if so can you share your solution? cheers.
